manim/mobject/mobject.py

```diff
@@ -1778,11 +1806,26 @@ def __init__(self, *mobjects, **kwargs):
 class _AnimationBuilder:
     def __init__(self, mobject):
         self.mobject = mobject
-        self.overridden_animation = None
         self.mobject.generate_target()
+
+        self.overridden_animation = None
         self.is_chaining = False
         self.methods = []
 
+        self.cannot_call = False
```

Comment:
A more descriptive name and/or a comment would be helpful for this variable.

Reply:
Is 28dda2a68a7ca338c9eda27c470d6a10ed1f3480 satisfactory?